src/commands/registries/azure/deployImageToAzure.ts

```diff
@@ -55,11 +60,64 @@ export async function deployImageToAzure(context: IActionContext, node?: RemoteT
+    if (node instanceof AzureTagTreeItem) {
+        let siteClient = new SiteClient(site, node.parent.parent.parent.root);
+        let appUri: string = (await siteClient.getWebAppPublishCredential()).scmUri;
+        await createWebhookForApp(node, wizardContext, appUri);
+    } else if (node instanceof DockerHubTagTreeItem) {
+        // point to dockerhub to create a webhook
+        // http://cloud.docker.com/repository/docker/<registryName>/<repoName>/webHooks
+        const dockerhubPrompt: string = "Copy web app endpoint and browse to dockerhub";
+        let response: string = await vscode.window.showInformationMessage("Please browse to your dockerhub account to set up a CI/CD webhook", dockerhubPrompt);
+        if (response) {
+            let appUri = `https://${site.name}.scm.azurewebsites.net/docker/hook`;
+            // tslint:disable-next-line:no-unsafe-any
+            await vscode.env.clipboard.writeText(appUri);
+            // account.name, repository.name
+            // tslint:disable-next-line:no-unsafe-any
+            vscode.env.openExternal(vscode.Uri.parse(`https://cloud.docker.com/repository/docker/${node.parent.parent.parent.username}/${node.parent.repoName}/webHooks`));
+        }
+    }
+
     ext.outputChannel.appendLine(createdNewWebApp);
     // don't wait
     window.showInformationMessage(createdNewWebApp);
 }
 
+async function createWebhookForApp(node: AzureTagTreeItem, wizardContext: IActionContext & Partial<IAppServiceWizardContext>, appUri: string): Promise<void> {
+    // fields derived from the app service wizard
+    let siteName: string = wizardContext.site.name;
+    let webhookName: string = `webapp${siteName}`;
+
+    // variables derived from the container registry
+    const crmClient = createAzureClient(node.parent.parent.parent.root, ContainerRegistryManagementClient);
+    let registryResourceGroupName: string = node.parent.parent.resourceGroup;
+    let registryName = node.parent.parent.registryName;
+
+    const registryList = await crmClient.registries.list();
+    const registryHandle = registryList.find((value) => value.name === registryName);
+    if (!registryHandle) {
+        return;
+    }
+    let webhookLocation: string = registryHandle.location;
+
+    const existingWebhooks = await crmClient.webhooks.list(registryResourceGroupName, registryName);
+    if (existingWebhooks.find((hook) => hook.name === webhookName)) {
+        return;
+    }
+
+    let webhookCreateParameters: WebhookCreateParameters = {
+        location: webhookLocation,
+        serviceUri: appUri,
+        scope: `${node.parent.repoName}:${node.tag}`,
+        actions: ["push"],
+        status: 'enabled'
+    };
+
+    const webhook = await crmClient.webhooks.create(registryResourceGroupName, registryName, webhookName, webhookCreateParameters);
+    ext.outputChannel.appendLine(`Created webhook ${webhook.name} with scope ${webhook.scope}, id: ${webhook.id} and location: ${webhook.location}`);
```

Comment:
Again - can you post a screenshot so that I can see what this message looks like?

Also generally wrap any dynamically inserted things (like `${webhook.name}`) in quotes

Reply:
![outputwebhook](https://user-images.githubusercontent.com/30420973/60471834-2c223480-9c1b-11e9-840c-24bbaab3601d.png)